diff --git a/src/ParseGeom.h b/src/ParseGeom.h
--- a/src/ParseGeom.h
+++ b/src/ParseGeom.h
@@ -47,6 +47,11 @@
 ******************************************************************/
 
+// from Xfuncproto.h
+#ifndef _Xconst
+#define _Xconst const
+#endif /* _Xconst */
+
 /* 
  * Bitmask returned by XParseGeometry().  Each bit tells if the corresponding
  * value (x, y, width, height) was found in the parsed string.
